sirepo/raydata/scan_monitor.py

```diff
@@ -254,8 +255,15 @@ async def _incoming(self, body):
         return getattr(self, "_request_" + body.method)(body.data.get("args"))
 
     async def post(self):
+        self._rs_authenticate()
         self.write(await self._incoming(PKDict(pkjson.load_any(self.request.body))))
 
+    def _rs_authenticate(self):
+        t = super()._rs_authenticate()
+        if t == sirepo.feature_config.for_sim_type("raydata").scan_monitor_api_secret:
```

Comment:
SIM_TYPE?

Reply:
I'm not sure where that would come from. Just set a variable on the module?

Reply:
Yes, "raydata" occurs in three separate places. It seems like this should be a constant in the module.

Reply:
014b4042b . Still trying to learn when you like to use const's:)

Reply:
I guess this is a taste thing. I think in this case, we have a model to name SIM_TYPE so that's why I recommended it. One case I often like consts is when there's no assertion on the string.